.github/workflows/registry-home-ci.yaml

```diff
@@ -0,0 +1,97 @@
+name: Registry Home UI CI
+
+on:
+  pull_request:
+    paths:
+      - "web/registry-home/**"
```

Comment:
registry-home vs business-registry-home ? I think it makes sense to call this business registry home everywhere right? It won't have the context of the repo name in the 1pass config or firebase so this will be a bit confusing